content/en/continuous_integration/tests/javascript.md

```diff
@@ -332,6 +332,14 @@ If the browser application being tested is instrumented using [RUM][6], your Cyp
 
 {{< /tabs >}}
 
+### Reporting code coverage
+
+When code coverage is available, the Datadog Tracer (v3.19.0+) reports it under the `test.code_coverage.lines_pct` tag for your test sessions.
+
+To compute the code coverage, you can use the `--coverage` flag in Jest or run your tests using [Istanbul][19] in other testing frameworks.
```

Comment:
I wouldn't single out `jest`. Maybe something like this: 

Also, let's not merge this until we're sure v3.19.0 is going to include the tag renaming 
```suggestion
When tests are instrumented with code coverage with [Istanbul][19], the Datadog Tracer (v3.19.0+) reports it under the `test.code_coverage.lines_pct` tag for your test sessions.
```

Reply:
Works for me, I mentioned Jest because Istanbul is built-in there, so the way to enable it is slightly different, but no strong opinion.